ethereum/messages.py

```diff
@@ -186,13 +188,19 @@ def apply_transaction(state, tx):
     null_sender = tx.sender == b'\xff' * 20
     if casper_contract and vote and null_sender:
         log_tx.debug("Applying CASPER VOTE transaction: {}".format(tx))
-        return _apply_casper_vote_transaction(state, tx)
+        return _apply_casper_no_gas_transaction(state, tx)
     else:
         log_tx.debug("Applying transaction (non-CASPER VOTE): {}".format(tx))
         return _apply_transaction(state, tx)
 
 
-def _apply_casper_vote_transaction(state, tx):
+def _apply_casper_no_gas_transaction(state, tx):
```

Comment:
`_method` should only be used in python for "internal" methods in a class or at least a file scope. Because this is not being used in `consensus.py` it should be renamed without underscore.